plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/quickfix/ChangeToStarProjectionFix.kt

```diff
@@ -82,6 +85,9 @@ class ChangeToStarProjectionFix(element: KtTypeElement) : KotlinQuickFixAction<K
             return null
         }
 
+        private fun Diagnostic.isArrayInstanceCheck(): Boolean =
+            factory == Errors.CANNOT_CHECK_FOR_ERASED && Errors.CANNOT_CHECK_FOR_ERASED.cast(this).a.isArrayOrNullableArray()
```

Comment:
Because the new fix is available only on jvm platform, we should leave this fix on non-jvm platforms, so at least some fix is available on every platform.

I guess it's good to leave a comment here to explain that we are depending on availability of `ConvertToIsArrayOfCallFix`.

Reply:
[add46db](https://github.com/JetBrains/intellij-community/pull/2083/commits/add46dbcca6ac0dc9c892ef1e080ad09a43b5283)